packages/cloud_firestore/example/lib/main.dart

```diff
@@ -42,7 +42,7 @@ class MessageList extends StatelessWidget {
           itemBuilder: (_, int index) {
             final DocumentSnapshot document = snapshot.data.documents[index];
             return ListTile(
-              title: Text(document['message'] ?? '<No message retrieved>'),
+              title: Text('${document['message']}' ?? '<No message retrieved>'),
```

Comment:
I don't think is quite what you want, we want to show '<No message retrieved>' if the message is `null`.